platform/duration.cpp

```diff
@@ -84,7 +84,7 @@ std::string Duration::GetString(std::initializer_list<Units> units, std::string_
   ASSERT(units.size(), ());
   ASSERT(IsUnitsOrderValid(units), ());
 
-  if (m_seconds.count() == 0)
+  if (SecondsToUnits(m_seconds, Units::Minutes) == 0)
```

Comment:
Is there a test for this case?

Reply:
No there are no tests for this case. I'll add one